clang/lib/Serialization/ASTWriter.cpp

```diff
@@ -237,6 +238,8 @@ GetAffectingModuleMaps(const Preprocessor &PP, Module *RootModule) {
     CollectIncludingMapsFromAncestors(CurrentModule);
     for (const Module *ImportedModule : CurrentModule->Imports)
       CollectIncludingMapsFromAncestors(ImportedModule);
+    for (const Module *UsedModule : CurrentModule->DirectUses)
```

Comment:
I don't really understand what this change does, and all the tests pass without it. Is it related to the rest of this patch?

(Looking at the other collections that happen here, it does seem to "fit". But I don't have a really clear idea about this piece).